### DISCUSSION TITLE
When a TinGS station is not connected to Internet, will received packets be lost?

### DISCUSSION THREAD

**Discussion by waterwin** [4m1g0/tinygs-wiki#7] [Q&A]
I want to test other locations for better reception but as I will be mobile I will not have Internet connection. It might even be without reach of 3G/4G so I can not use a smartphone AP via 4G.

I will plan it based on gpredict data for location of satellite footprint. So I expect to receive packets.

Thanks
de Erwin

**Comment by judhi** [comment]
Better late than never :)

You can log the output from serial port to a computer and retrieve that later.
